src/test/java/org/takes/rs/RsWithHeaderTest.java

```diff
@@ -62,4 +63,18 @@ public void addsHeadersToResponse() throws IOException {
         );
     }
 
+    /**
+     * RsWithHeader can't add invalid headers.
+     * @throws IOException If some problem inside
+     */
+    @Test(expected = IllegalArgumentException.class)
+    public void notAddsInvalidHeadersToResponse() throws IOException {
+        final Iterable<String> head = new RsWithHeader(
+            new RsWithHeader(new RsEmpty(), "host:", "c.example.com"),
+            "Host MY", "d.example.com"
+        ).head();
+        for (final String header: head) {
```

Comment:
as far as I understand, this code will never be executed. we'll fail at the previous line. correct? if so, why do we need it?